packages/web-components/src/common/utils/map-helper.ts

```diff
@@ -0,0 +1,197 @@
+import { PropertyModel } from '@reapit/foundations-ts-definitions'
+import { INVALID_BACKGROUND_AS_BASE64, DEFAULT_CENTER } from './constants'
+import { SearchType } from '../../search-widget/client/core/store'
+
+export const getLatLng = (property: PropertyModel) => {
+  const latitude = property?.address?.geolocation?.latitude
+  const longitude = property?.address?.geolocation?.longitude
+  return {
+    latitude,
+    longitude,
+  }
+}
+
+export type GetContentParams = {
+  latitude?: number
+  longitude?: number
+  address: {
+    line1: string
+    line2: string
+  }
+  bedrooms?: number
+  bathrooms?: number
+  marketingMode?: string
+  lettingPrice?: number
+  rentFrequency?: string
+  imageUrl?: string
+  price: string
+}
+
+export const getContent = ({
```

Comment:
Does this not make sense as a svelte component? I get why you wouldn't when it was a React component but Svelte is just HTML and CSS - we could get rid of the inline styles too